libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/camera/Camera.kt

```diff
@@ -0,0 +1,32 @@
+package com.mapbox.services.android.navigation.v5.navigation.camera
+
+import com.mapbox.geojson.Point
+
+/**
+ * This class handles calculating camera's zoom and tilt properties while routing.
+ * The [com.mapbox.services.android.navigation.v5.navigation.MapboxNavigation] uses
+ * a [SimpleCamera] by default. If you would like to customize the camera properties, create a
+ * concrete implementation of this class or subclass [SimpleCamera] and update
+ * [com.mapbox.services.android.navigation.v5.navigation.MapboxNavigation.setCameraEngine].
+ *
+ * @since 0.10.0
```

Comment:
NIT This can be removed

Reply:
I was referring to `@since 0.10.0` line.

Reply:
This is already removed. I cannot see it in my local. 😛